### PR TITLE
Move adr settings to flows node info only

### PR DESCRIPTION
**Pull request by vicmgs**
anything related to telling router about device adr status will be worked on later

### CODE REVIEW

**Comment by ambuznego** [comment]
perhaps different PR but show page doesn't show  anything about ADR for devices

File: assets/js/components/flows/infoSidebar/AdrNodeSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        onChange={adrValue => updateAdr(adrValue)}
+        checked={checked}
+        style={{ marginRight: 8 }}
+      />
+      <Text strong style={{ fontSize: 16 }}>Allow ADR (recommended for stationary devices)</Text>
+    </div>
+
+    <div style={{ marginBottom: 20 }}>
+      <Text style={{ fontSize: 14 }}>{
+        "Adaptive Data Rate (ADR) needs to be requested by a device for this setting to have an effect. ADR allows devices to use an optimal data rate which reduces power consumption and airtime on the network based on RF conditions. However, it is recommended to only use this setting for fixed or non-mobile devices to ensure reliable connectivity."
```
Comment (ambuznego) [inline assets/js/components/flows/infoSidebar/AdrNodeSettings.jsx:18]: wonder if we could reuse this text from what's on the adr index

**Review by ambuznego — APPROVED** [review]
just a few minor things

**Comment by vicmgs** [comment]
will add adr to show page later

**Pull request merged**